scalding-core/src/main/scala/com/twitter/scalding/serialization/CascadingBinaryComparator.scala

```diff
@@ -53,6 +54,7 @@ object CascadingBinaryComparator {
 
     def check(s: Splice): Try[Unit] = {
       val m = s.getKeySelectors.asScala
+      val sortingSelectors = s.getSortingSelectors.asScala
```

Comment:
We can, actually I had a note here to ask you if you think this should be a separate option entirely? Given we never really expose the key sorting ability to users in a real manner just using our sorting should be fine for them. But secondary sort is where you may/probably want to customize ?